README.md

```diff
@@ -151,6 +151,13 @@ File --> Settings --> Editor --> Code Style --> JSON --> Wrapping and Braces
 | Objects| Wrap if long
 | Align| Do not align
 
+## visualizer
```

Comment:
Englisch würde mir besser gefallen (da die README an sich Englisch ist). Außerdem weiter oben in der README eine kleine Section wie man den BrainVisualizer ausführt (visualize.py --neuron-vis und es ist wichtig --render mitanzugeben)